spharpy/samplings/coordinates.py

```diff
@@ -310,3 +348,33 @@ def n_max(self, value):
             self._n_max = None
         else:
             self._n_max = int(value)
+
+    @property
+    def quadrature(self):
+        """Get or set the quadrature flag."""
+        return self._quadrature
+
+    @quadrature.setter
+    def quadrature(self, value):
+        """Get or set the quadrature flag."""
+
+        # check input
+        if not isinstance(value, bool):
+            raise TypeError(
+                f'quadrature must be True or False but is {value}')
+
+        # flag indicating if weights sum to 4 pi
+        if self.weights is None:
+            weights_sum = False
+        else:
+            weights_sum = np.abs(np.sum(self.weights) - 4 * np.pi) < \
```

Comment:
not sure if this should be tested here or in the sh class, I guess @mberz can clarify

Reply:
see #123